packaging/language/pip.py

```diff
@@ -273,7 +273,7 @@ def _get_pip(module, env=None, executable=None):
     # interpreter, so people can use pip to install modules dependencies
     candidate_pip_basenames = ['pip2', 'pip', 'python-pip', 'pip-python']
     if PY3:
-        candidate_pip_basenames = ['pip3']
+        candidate_pip_basenames = ['pip3', 'pip']
```

Comment:
I'm not sure about this.  Linux distros are definitely making /usr/bin/pip the python2 version.  Adding pip here will mean that a site that has python2-pip installed from system packages but python3-pip is not installed will end up using the python2 version even though the python3 version is expected.  We should return an error to the user in that case instead.